railties/lib/rails/commands/test.rb

```diff
@@ -1,9 +1,19 @@
-require "rails/test_unit/minitest_plugin"
+require 'rails/test_unit/minitest_plugin'
+require 'rails/commands/command'
 
-if defined?(ENGINE_ROOT)
-  $: << File.expand_path('test', ENGINE_ROOT)
-else
-  $: << File.expand_path('../../test', APP_PATH)
-end
+module Rails
+  module Commands
+    # This is a wrapper around all Rails test tasks, including:
+    #   rails test
+    #   rails test:db
+    class Test < Command
+      def test
+        system("bin/rake test")
```

Comment:
The idea is to move away from Rake, so this is going the wrong way (sample for demonstration or not).